### DISCUSSION TITLE
RPI5 NVMe

### DISCUSSION THREAD

**Discussion by WirthmU** [aaronwmorris/indi-allsky#1539] [General]
Does anyone already have experience running an RPI5 including indi-allsky entirely from NVMe, i.e. instead of booting from the SD card to boot from the NVMe and have the SD card as second in chain boot device?

**Comment by chrisramsay** [comment]
I'm running it entirely from nvme. Bought and used a Pimoroni module and followed the steps towards the end of this guide: https://learn.pimoroni.com/article/getting-started-with-nvme-base
